### ISSUE TITLE
All logic and styles are contained in the index.html file

### ISSUE THREAD

**Issue by ChristofLee** [nirmankarta/eddyst.one#8]
I would suggest moving the custom CSS and JS to their own files.

Then using the Gulp task created in #7 to minify/uglify these files ready for production.

**Comment by PrabhanshuAttri** [comment]
@ChristofLee I was thinking of creating this issue. Could you please separate styles and js code to separate files. After this, we can merge gulp minification as well.

**Comment by Captain-DarkO** [comment]
shall I?

**Comment by PrabhanshuAttri** [comment]
@DeeJay-NuKe This issue is already assigned to @ChristofLee. Explore the webpage and let us know if something needs to be added to the page.

**Comment by Captain-DarkO** [comment]
okay 👍

**Comment by ChristofLee** [comment]
I can do the work based on #6 . 

However I think it would be best to review, test and merge my PR #6 and then review this issue as a separate PR.

I think it would make it easier for you to review my changes individually, instead of one huge PR.

It's up to you, just a suggestion.

**Comment by ChristofLee** [comment]
I have submitted a PR for this #10 but it may contain conflicts because it waits on #6 . 

Please let me know if you need anything more from me 👍 

There are some enhancements that could be implemented to the CSS and JS but they could/should be raised as issues to let more people get a chance to create features.

**Comment by PrabhanshuAttri** [comment]
Thanks @ChristofLee. Good going. As #6 is assigned to someone else, could you please just separate the CSS and JS. @fatimarafiqui can work on gulp task.

Feel free to open issues. :)

**Comment by fatimarafiqui** [comment]
@PrabhanshuAttri @ChristofLee I am working on the minification issue.

**Comment by ChristofLee** [comment]
This is highly dependent on the file structure and the way the minification is implemented.

I would need to wait and see how @fatimarafiqui implements it before I could make a viable Pull Request.

Another issue, @tommyka has created a PR on top of #6 to add browsersync. At this point it would be faster and easier to use my PR #7 and allow @fatimarafiqui to make changes as they see fit.

Also, if you are only allowing the assigned person to work on the issue, please assign the ticket to them.

**Comment by PrabhanshuAttri** [comment]
@ChristofLee and @tommyka  good to see you guys collaborate. I didn't expect this amount of response for this repository that's why I was not assigning the issues. It was expected that the contributor would read the issues and comments before creating the PR.

Don't worry your PR won't get rejected. That said, @fatimarafiqui PR looks good. I will merge her PR and then @ChristofLee you can just rebase your 2 pending PRs with all missing features/fixes. I believe this would be fair to all. 

It is advised to discuss before adding features.
Thanks @ChristofLee @fatimarafiqui and @tommyka

**Comment by PrabhanshuAttri** [comment]
@ChristofLee @fatimarafiqui and @tommyka Feel free to contribute to [PyBeacon](https://github.com/nirmankarta/PyBeacon) as well. Eddyst.one is just a side project for PyBeacon.

**Comment by ChristofLee** [comment]
@PrabhanshuAttri That sounds good to me (and fair) 👍  I've been doing Hacktoberfest, and I had free time yesterday so was just trying to get my PR's in ;) (couldn't wait until today).

I have made another suggestion #16 which might make things easier. 

Will check out PyBeacon too. If there is anything specific you want me to take a look at, tag me in 👍 

Something I have done in the past with open source projects is create a [Gitter](https://gitter.im/) channel. This would help communication and the discussion of ideas. Might be a route to consider?

@PrabhanshuAttri @fatimarafiqui If your PR can be finished within the next 2 days, I will do like you suggested and rebase my changes. Unfortunately, after that I won't be available 👎

**Comment by PrabhanshuAttri** [comment]
@ChristofLee I understand. As far as I know, PR doesn't have to be merged. We do have a [slack](http://nirmankarta.slack.com) channel for communication. You can invite yourself using this [link](http://nirmankarta.herokuapp.com). Please make changes to your PR so that I can review and merge them.

**Comment by ChristofLee** [comment]
@PrabhanshuAttri I think there may be an error on that link.
>Failed! Something has gone wrong. Please contact a system administrator.

**Comment by PrabhanshuAttri** [comment]
Try again

**Comment by ChristofLee** [comment]
Nope, still receiving that same error message 👎

**Comment by PrabhanshuAttri** [comment]
Could you please specify which link? I must have misunderstood.

**Comment by ChristofLee** [comment]
Sorry, after clicking the link, I enter my email, and get the error message. Please see screenshot.

http://nirmankarta.herokuapp.com/

![image](https://user-images.githubusercontent.com/4659534/32273533-b4f3811a-befa-11e7-8313-6b2681318116.png)

**Comment by PrabhanshuAttri** [comment]
I am not sure what's the problem here as it works for me.
![selection_039](https://user-images.githubusercontent.com/4739486/32273588-09b0fbdc-bf29-11e7-8e6a-17b8e1fcd8da.png)

**Comment by PrabhanshuAttri** [comment]
Try now.
